tests/test_models.py

```diff
@@ -117,10 +138,20 @@ def test_balance_excludes_failed_transactions(self):
             quantity=-55,
             state=models.TransactionStateChoices.FAILED,
         )
-        expected_balance = 100 - 10 - 10 - 10 + 10 - 10
+        expected_balance = 100 - 10 - 10 - 10 + 10 - 10 + 15 - 5 + 50
 
         self.assertEqual(self.ledger.balance(), expected_balance)
 
+    def test_total_deposits(self):
+        """
+        Test Ledger.total_deposits() counts all deposit-esque transactions.
+        """
+        #                       extra deposit─────────────────┐
+        #                         adjustments────────┬────┐   │
+        #                     initial deposit───┐    │    │   │
+        assert self.ledger.total_deposits() == 100 + 15 - 5 + 50
```

Comment:
This is a very cool visual you added. Wondering if you used a tool to generate this or was it all by hand?

Reply:
I used plain old vim! I grab the characters from https://en.wikipedia.org/wiki/Box-drawing_characters#Unix,_CP/M,_BBS and use vim's standard rectangular editing modes to manipulate the characters.